resources/wikia/ui_components/modal/js/modal.js

```diff
@@ -173,7 +174,7 @@ define('wikia.ui.modal', [
 		this.$element = $(jQuerySelector);
 		this.$content = this.$element.children('section');
 		this.$close = this.$element.find('.' + CLOSE_CLASS);
-		this.$blackout = $('#' + blackoutId);
+		this.$blackout = $('#' + blackoutId).addClass(BLACKOUT_HIDDEN_CLASS);
```

Comment:
What do you think about having this class added by default in the template and then just removing it with jQuery?